spec/features/map/granule_hole_display_spec.rb

```diff
@@ -14,7 +14,8 @@
       end
 
       it 'draws footprints that contain holes' do
-        expect(page).to have_selector('.leaflet-overlay-pane path', count: 14)
+        # expect(page).to have_selector('.leaflet-overlay-pane path', count: 14)
```

Comment:
I know some things changed with layers, but this raises a flag for me.. Is there a known reason these counts are less?